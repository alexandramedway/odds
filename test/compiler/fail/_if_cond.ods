do if 42 then true else false
