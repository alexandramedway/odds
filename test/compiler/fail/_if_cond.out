Analyzer Exception: Expected type Bool but got type Num instead
